src/secex-genprotimgvm-scripts/post-script.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+echo "Moving sdboot and executing zipl"
+
+workdir="/build"
+sdboot="/genprotimg/se.img"
+genprotimg_dir="${workdir}/genprotimg"
+se_boot="/tmp/se"
+
+disk=$(realpath /dev/disk/by-id/virtio-target)
+disk_se="${disk}1"
+
+mkdir "${se_boot}"
+mount "${disk_se}" "${se_boot}"
+cp "${sdboot}" "${se_boot}/sdboot"
+zipl -V -i ${se_boot}/sdboot -t ${se_boot}
```

Comment:
I've added it since it makes the script look nicer.
I'm not sure if it is really needed, as the VM is readonly and shuts down after this script anyway.